releases/v0.13.4.yaml

```diff
@@ -0,0 +1,7 @@
+---
+version: v0.13.4
+name: 0.13.4
+branch: release-0.13
+status: shipyard
+components:
+  shipyard: 7b46174780e75440013788cd2858aca35b8d5ae0
```

Comment:
```suggestion
  shipyard: a3eee257fc83022342b49c0b959a17d29457f33e
```

Reply:
It's surprising that re-running the `make release` command doesn't result in updating this